Core/Core/API/APIRecentlyGradedRequestable.swift

```diff
@@ -0,0 +1,34 @@
+//
+// Copyright (C) 2019-present Instructure, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, version 3 of the License.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+//
+
+import Foundation
+
+public struct GetRecentlyGradedSubmissionsRequest: APIRequestable {
+    public typealias Response = [APISubmission]
+
+    let userID: String
+
+    public var path: String {
+        return "users/\(userID)/graded_submissions"
```

Comment:
Nitpick: we've been using `ContextModel` for these sorts of paths.

```swift
let context = ContextModel(.user, id: userID)
return "\(context.pathComponent)/graded_submissions"
```

Reply:
Oh I knew about this but didn't think of user as a context. I will swap this.